### DISCUSSION TITLE
What's under the hood of the `cors-proxy` that's used on `getlang.dev`?

### DISCUSSION THREAD

**Discussion by ilyazub** [getlang-dev/get#21] [General]
Thank you for the cool project!

What's used under the hood of the `cors-proxy` endpoint? (Response time is pretty good &mdash; below 1.5 seconds for Google SERPs with `num=100`.)

Thank you.

**Comment by mattfysh** [comment]
Hi @ilyazub - there are a list of open cors proxies here: https://gist.github.com/jimmywarting/ac1be6ea0297c16c477e17f8fbe51347

getlang.dev use just two, falling back to the second if the first fails. These are corsproxy.io and cors-proxy.fringe.zone

One downside is that they do cache aggressively, so be mindful of which headers you're sending. Cheers
